packages/solana-wallet-adapter/src/SolanaWalletAdapterConfig.tsx

```diff
@@ -65,19 +65,20 @@ function SolanaWalletAdapterConfig({
   testnetChains?: Chain[];
 }): JSX.Element | null {
   // For now support only a single chain
-  const { setChains } = useChain<
+  const { setChains, selectedChain } = useChain<
     SupportedChains.Solana,
     Chain & BaseChain,
     never
   >();
   const endpoint = useMemo(() => {
-    if (chains.length === 0) {
+    if (chains.length === 0 || selectedChain?.rpcEndpoint === undefined) {
```

Comment:
Why not just `!selectedChain?.rpcEndpoint` ?
Are we sure it can't be `null` , fpr e.g. ?

Reply:
fixed